packages/instant-meilisearch/src/adapter/search-request-adapter/search-params-adapter.ts

```diff
@@ -176,15 +173,26 @@ export function MeiliParamsCreator(searchContext: SearchContext) {
         meiliSearchParams.sort = [sort]
       }
     },
-    addGeoSearchRules() {
-      const geoSearchContext = createGeoSearchContext(searchContext)
-      const geoRules = adaptGeoPointsRules(geoSearchContext)
+    addGeoSearchFilter() {
```

Comment:
Same here, I'm confused with the naming. 
Upper you do the following:
`import { adaptGeoFilter } from './geo-rules-adapter'`
And here you're writing `addGeoSearchFilter`. In the code you sometimes have `geoXXX`, `geoPointsXXX` and `geoSearchXXX`. I feel like the `geoXXX` is no precise enough and we don't really know what it is exactly. 
Is this intended or is something missing in the naming?